docs/application/dotnet/guides/nui/text.md

```diff
@@ -342,6 +352,15 @@ The following markup elements are currently supported:
 
 
 
+### Use Markup to represent encoded characters
+
+Markup text is not allowed to contain some characters unless they are representing tags such as "<" or ">". To use these characters or represent any unicode character, you must use reserved entities.

```

Comment:
@surbhig12 I thought according to the SRI-B style, we need a sentence here.

The following example uses reserved entities:

Would you please check the sentence?

Reply:
Suggestion:

Markup text is not allowed to contain some characters unless they are representing tags such as "<" or ">". To use these characters or represent any unicode character, you must use reserved entities. The following example uses reserved entities:

Reply:
@surbhig12 Your suggestion has no big difference with mine, so I hope you to check again it there wasn't a mistake.

Reply:
@mijin85cho, the intention was not to change the sentence completely. I agree with your suggestion of "The following example uses reserved entities:", therefore I added it as a followign sentence.

Reply:
Thanks for the reply, @surbhig12 

@Ali-Alzyoud as there were some missed points, would you please update them also? After that I'll ask @surbhig12 to check and approve again to proceed to next step.